_posts/2020-11-27-changelog-136.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #136"
+date: 2020-11-27 09:07:11 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6378 -->
+* Introduce new experimental engine for name resolution (disabled by default).
+  The new algorithm (aka name resolution 2.0) uses the fixed-point iteration approach and aims at better support for
+  complex glob imports (`use foo::*`) and re-exports. Also, it will allow the plugin to properly expand the up-coming
+  macros 2.0. Note that the new algorithm works for top-level items only, so local variables, as well as type inference
+  functionality, won’t be affected. For now, the newer version of the algorithm is available as an experimental feature.
```

Comment:
```suggestion
  functionality, won’t be affected.
```

"experimental feature" is a concrete term in our IDEs as a result, it's not true since we introduce user-visible option to enable it. Let's just drop this sentence